Add New Chaplain / Religious Equipment and Items

XanderDox opened the pull request:
This PR does the following!:

**1. Adds/Ports New Religious Accessories**
The following have been ported from Aurora and added to the loadout:

- Gold and silver crucifix necklaces
- Gold and silver upside-down crucifix necklaces
- Scapular
- Rosary (can be used in-hand for special interaction to count beads in prayer)
- The Jewish tallit prayer-shawl has been ported from Aurora and added to the loadout.

Adds:
- Generic prayer beads into loadout that are customizable.

**2. Adds New Chaplain Equipment**
The Chaplain's Locker will now include a 'Christianity Supplies Box', which includes:
- An aspergillum, allowing them to sprinkle people with ~polytrinic acid~ holy water during ceremonies, it even jingles.
- Two Holy Icons, one of Jesus Christ, one of the Blessed Virgin Mary.
- A new cassock based on a modified Aurora sprite.
- A new clergy collar.
- Two silver crucifix necklaces for gifting to crew.
- One scapular for personal use or gifting to crew.
- One rosary for personal use or gifting to crew.
- A nun's habit and hood.

<img width="374" height="270" alt="image" src="https://github.com/user-attachments/assets/85ef676c-d32b-4b01-ad3e-e6cc10cb964e" />
<img width="159" height="182" alt="image" src="https://github.com/user-attachments/assets/d01561ec-05dc-474f-a468-3174d2bed4cb" />
<img width="168" height="182" alt="image" src="https://github.com/user-attachments/assets/8a46c8ba-a939-4c57-a54d-d63e57c49169" />


**What's Next?**
This is the first PR in what I hope to make a series of small chaplain item expansions. For major faiths, I hope to create more supplies boxes so that chaplains can feel equipped to setup the chapel to reflect their chosen religion and have trinkets to engage crew in their activities. Eventually, this will probably be shifted towards chaplains spawning with their respective supplies boxes based on their selected faiths.

:cl: XanderDox
rscadd: Adds a box of Christianity Supplies full of new items to the Chaplain Locker, which includes: Aspergillum, two Holy Icons, a cassock, a clergy collar, silver crucifix necklaces, a scapular, rosary, and nun's habit and hood.
rscadd: Ports a rosary. Add functionality to it so that it can be interacted with to count beads in prayer for fluff.
rscadd: Ports a Jewish tallit/prayer shawl.
rscadd: Adds generic prayer beads and puts them in the loadout.
imageadd: Ports sprites from Aurora for gold and silver crucifixes, scapulars, rosaries, aspergillum, and cassock.
soundadd: Ports the jingle sound from Aurora for the aspergillum.
imageadd: Adds a sprite of a Holy Icon of Mary.
imageadd: Adds a sprite for a large box of Christianity Supplies.
imageadd: Adds greyscale sprites for prayerbeads, courtesy of Azlan.
/:cl:

HeyBanditoz commented on code/modules/clothing/under/accessories/holywear.dm:
```diff
@@ -0,0 +1,79 @@
+			user.visible_message("\The [user] finishes counting through the beads of \the [src].", "You finish counting through the beads of \the [src].")
+			for(var/mob/living/carbon/human/H in view(user))
+				if(user.get_cultural_value(TAG_RELIGION) == H.get_cultural_value(TAG_RELIGION))
+					to_chat(H, SPAN_NOTICE("You feel calm and relaxed, and at one with God."))
```
Some style stuff (untested)
```suggestion
/obj/item/clothing/accessory/rosary/attack_self(mob/living/carbon/human/user)
	if (!ishuman(user) || !user.mind)
		return
	user.visible_message("\The [user] begins to count through the beads of \the [src] while muttering prayers...", "You begin to count through the beads of \the [src] while muttering prayers...")
	if (!do_after(user, 5 SECONDS, src, do_flags = DO_PUBLIC_UNIQUE))
	    return
	user.visible_message("\The [user] finishes counting through the beads of \the [src].", "You finish counting through the beads of \the [src].")
	for (var/mob/living/carbon/human/human_in_view in view(user))
		if (user.get_cultural_value(TAG_RELIGION) == human_in_view.get_cultural_value(TAG_RELIGION))
			to_chat(human_in_view, SPAN_NOTICE("You feel calm and relaxed, and at one with God."))
```

HeyBanditoz replied:
Superseded by Cakey's comments

HeyBanditoz commented on code/__defines/lighting.dm:
```diff
@@ -86,6 +86,7 @@
 #define LIGHT_COLOUR_E_RED	"#da0205" // Emergency red lighting
 #define LIGHT_COLOUR_READY	"#00ff00" // Green 'READY' lighting
 #define LIGHT_COLOUR_SKRELL	"#66ccff" // Skrellian cyan lighting
+#define LIGHT_COLOR_HALOGEN	"#C0C0CA" // Standard halogen light color
```
nit: match incorrect :^) spelling of color, and lowercase hex code
```suggestion
#define LIGHT_COLOUR_HALOGEN	"#c0c0ca" // Standard halogen light color
```

HeyBanditoz commented on code/game/objects/items/weapons/material/misc.dm:
thoughts on changing the names here to lowercase, i.e. `icon of Christ` otherwise the game will think it's a proper noun:
`Iris Hanford has thrown Icon of Christ.` would become `Iris Hanford has thrown the icon of Christ.`

HeyBanditoz replied:
alternatively, add `\improper` to the start of the name, like `name = "\improper Icon of Christ"` which looks better than what I recommended above

XanderDox replied:
I will fix this up! Still getting used to not capitalizing things.

HeyBanditoz commented on code/modules/clothing/under/accessories/holywear.dm:
```diff
@@ -0,0 +1,79 @@
+
+/obj/item/clothing/under/cassock
+	desc = "It's a black cassock, often worn by Catholic clergy."
+	name = "clergy cassock"
```
this is gnawing at me :p
```suggestion
	name = "clergy cassock"
	desc = "It's a black cassock, often worn by Catholic clergy."
```

HeyBanditoz reviewed (dismissed):
1st quick look

CakeQ commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -728,3 +728,19 @@
 	)
+
+/obj/item/storage/box/large/christianity
+	name = "box of christian supplies"
```
capitalize Christian

CakeQ commented on code/modules/clothing/under/accessories/holywear.dm:
```diff
@@ -0,0 +1,79 @@
+	if(!ishuman(user))
+		return
+	if(user.mind)
+		user.visible_message("\The [user] begins to count through the beads of \the [src] while muttering prayers...", "You begin to count through the beads of \the [src] while muttering prayers...")
```
We don't want items that make broad assumptions of what a player or character would be doing beyond directly interacting with something for it's immediate purpose. The user of this rosary could be a non-Christian or even a different sect of Christian. If the player wants to be praying while doing this, they can do so themselves via emotes, speech, or even with the pray command. Please change this message to only mention counting beads.

CakeQ commented on code/modules/clothing/under/accessories/holywear.dm:
```diff
@@ -0,0 +1,79 @@
+		user.visible_message("\The [user] begins to count through the beads of \the [src] while muttering prayers...", "You begin to count through the beads of \the [src] while muttering prayers...")
+		if(do_after(user, 5 SECONDS, src, do_flags = DO_PUBLIC_UNIQUE))
+			user.visible_message("\The [user] finishes counting through the beads of \the [src].", "You finish counting through the beads of \the [src].")
+			for(var/mob/living/carbon/human/H in view(user))
```
This also falls under the above, where we don't want to take player control away from them in scenarios where they are capable of reacting themselves to what's happening. Please remove this code block.

CakeQ replied:
This remains unresolved

XanderDox replied:
To clarify, you don't want the beads to have any mechanical effect on-click for counting? I removed references to prayer. Little functionalities like this improve player experience and make them feel a bit more connected to the personal items they carry, the ability to click to count beads in no way prevents someone from using /me to count them as well.

CakeQ replied:
Yeah this should be kept to just bead counting, and not sending out additional messages to others.

CakeQ commented on code/__defines/lighting.dm:
```diff
@@ -86,6 +86,7 @@
 #define LIGHT_COLOUR_E_RED	"#da0205" // Emergency red lighting
 #define LIGHT_COLOUR_READY	"#00ff00" // Green 'READY' lighting
 #define LIGHT_COLOUR_SKRELL	"#66ccff" // Skrellian cyan lighting
+#define LIGHT_COLOUR_HALOGEN	"#c0c0ca" // Standard halogen light color
```
What is this for?

XanderDox replied:
Woops, that's a left-over from when I was attempting to include a projector thing with this. I'll pull that out.

CakeQ commented on code/modules/clothing/under/accessories/holywear.dm:
```diff
@@ -0,0 +1,91 @@
+	slot_flags = SLOT_BELT | SLOT_TIE
+
+/obj/item/clothing/accessory/prayer_beads/attack_self(mob/living/carbon/human/user)
+	if(!ishuman(user))
```
If prayer beads and rosaries share the same functionality then one should sub-class off of the other

i.e. /obj/item/clothing/accessory/prayer_beads/rosary

That way you don't have to re-define the same attack_self function

CakeQ commented on code/game/objects/items/weapons/material/misc.dm:
```diff
@@ -154,3 +154,17 @@
+	icon_state = "icon_christ"
+	w_class = ITEM_SIZE_SMALL
+
+/obj/item/icon_mary
```
Wasn't going to mention this one but with the other change request you might as well.

This should be a sub-class of icon_christ (i.e. /obj/item/icon_christ/mary) to reduce duplicate variables. That way you only need to replace the name/desc/icon_state

CakeQ reviewed (changes requested):
Final pass, sorry!

XanderDox committed:
Adds/Ports Religious Items

Adds a variety of religious items (cassock, icons, rosaries, prayer beads, scapulars, tallit, and crucifixes),  as well as a Christianity Supply Box in the Chaplain's locker. Adds rosaries, crucifixes, prayer beads and icons to the loadout.

XanderDox commented:
Fixed a build error that @HeyBanditoz pointed out with my supplies box being overfull. Removed some items from it, commits squashed.
